### PR TITLE
fix(curriculum): edited tutorial.md (stout.txt -> stdout.txt)

### PR DESCRIPTION
**Pull request by kalashjain23**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes <a href ="https://github.com/freeCodeCamp/freeCodeCamp/issues/44875">#44875</a>

### CODE REVIEW

File: tutorial.json [hunk 1]
```diff
@@ -3068,7 +3068,7 @@
           "hints": [
             "Here's an example: `cat $1 | sed 's/<pattern>/<replacement>/'`",
             "The `sed` argument should be `s/catnip/dogchow/`",
-            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow/'\n```"
+            "The `translate.sh` file should look like this:\r\n```sh\r\n#!/bin/bash\r\n\r\ncat $1 | sed 's/catnip/dogchow/'\r\n```"
```
Comment (moT01) [inline tutorial.json:3071]:
```suggestion
            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow/'\n```"
```

File: tutorial.json [hunk 2]
```diff
@@ -3169,7 +3169,7 @@
           "content": "It didn't output anything, so it must be replacing all the instances of `catnip`. You can replace many patterns using `sed` like this: `sed 's/<pattern_1>/<replacement_1>/; s/<pattern_2>/<replacement_2>/'`. Note that you need the semi-colon between the two replacement patterns and they both need to be wrapped in the quotes. In your script, add another pattern to the `sed` command that replaces `cat` with `dog`.",
           "hints": [
             "The code looks like this: `s/cat/dog/`",
-            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow/; s/cat/dog/'\n```"
+            "The `translate.sh` file should look like this:\r\n```sh\r\n#!/bin/bash\r\n\r\ncat $1 | sed 's/catnip/dogchow/; s/cat/dog/'\r\n```"
```
Comment (moT01) [inline tutorial.json:3172]:
```suggestion
            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow/; s/cat/dog/'\n```"
```

File: tutorial.json [hunk 3]
```diff
@@ -3245,7 +3245,7 @@
             "You can add another pattern to replace like before. Add a semi-colon and another pattern in the quotes of the `sed`",
             "Here's an example: `sed 's///; s///; s///'`",
             "The third pattern should be `s/meow/woof/`",
-            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/; s/cat/dog/; s/meow/woof/'\n```"
+            "The `translate.sh` file should look like this:\r\n```sh\r\n#!/bin/bash\r\n\r\ncat $1 | sed -E 's/catnip/dogchow/; s/cat/dog/; s/meow/woof/'\r\n```"
```
Comment (moT01) [inline tutorial.json:3248]:
```suggestion
            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/; s/cat/dog/; s/meow/woof/'\n```"
```

File: tutorial.json [hunk 4]
```diff
@@ -3323,7 +3323,7 @@
           "hints": [
             "Here's an example of one pattern: `s/<pattern>/<replacement>/<regex_flags>`",
             "It's the `g` flag",
-            "Your `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow/woof/g'\n```"
+            "Your `translate.sh` file should look like this:\r\n```sh\r\n#!/bin/bash\r\n\r\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow/woof/g'\r\n```"
```
Comment (moT01) [inline tutorial.json:3326]:
```suggestion
            "Your `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow/woof/g'\n```"
```

File: tutorial.json [hunk 5]
```diff
@@ -3372,7 +3372,7 @@
           "hints": [
             "The last `sed` pattern in you scrip should be `s/meow|meowzer/woof/`",
             "And you should add the `-E` flag to the `sed` command",
-            "Your `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow|meowzer/woof/g'\n```"
+            "Your `translate.sh` file should look like this:\r\n```sh\r\n#!/bin/bash\r\n\r\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow|meowzer/woof/g'\r\n```"
```
Comment (moT01) [inline tutorial.json:3375]:
```suggestion
            "Your `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow|meowzer/woof/g'\n```"
```

**Review by moT01 — COMMENTED** [review]
Thanks for the PR @kalashjain23 - I'm going to remove all the `\r` changes that were added. Not anything you could have avoided.

**Comment by moT01** [comment]
LGTM 🎉 Thanks for fixing this @kalashjain23 👍

**Comment by kalashjain23** [comment]
> LGTM 🎉 Thanks for fixing this @kalashjain23 👍

OMG!! Thank Youuu @moT01 !! This is my first PR that is approved. Thank youuu againnn😁😁🙌🙌

P.S : If you have some good issues for beginners, feel free to tag me there!! 😉

**Comment by moT01** [comment]
You can look through [issues on the freeCodeCamp repo](https://github.com/freeCodeCamp/freeCodeCamp/issues) if you want - any of them with the 'help wanted' label are open for contribution.

**Comment by kalashjain23** [comment]
> You can look through [issues on the freeCodeCamp repo](https://github.com/freeCodeCamp/freeCodeCamp/issues) if you want - any of them with the 'help wanted' label are open for contribution.

okkkk thankss😁🙌

**Pull request merged**